src/components/composites/Toast/Toast.tsx

```diff
@@ -271,15 +271,15 @@ export const ToastProvider = ({ children }: { children: any }) => {
           <Alert
             maxWidth="100%"
             alignSelf="center"
+            flexDirection="row"
             status={status ?? 'info'}
             variant={variant as any}
             accessibilityLiveRegion={accessibilityLiveRegion}
             {...rest}
           >
-            <VStack space={1} flexShrink={1} w="100%">
+            <VStack space={1} flexShrink={1}>
               <HStack
                 flexShrink={1}
-                space={2}
```

Comment:
Why did we removed the spacing here?

Reply:
it was creating a issue with info icon and close button. Both were not having same distance from text when title is long on IOS and Android. small UI change suggested by Himanshu.